### ISSUE TITLE
[Suggestion] waiting time option

### ISSUE THREAD

**Issue by HanatakeYurii** [Tonius/E-Mobile#7]
Hi,

I would like to suggest a config option for the time between the call and TP, 8 seconds sounds a bit long so it could be great if we can set for example 5 seconds in the configs.

**Comment by K4N0** [comment]
yes please and that if you take damage or move it stops the teleport. it's to prevent pvp logging.
I really love this mod!! Can i make a mod Showcase on it?
